app/lib/features/chat/widgets/chat_layout_builder.dart

```diff
@@ -16,19 +18,26 @@ class ChatLayoutBuilder extends StatelessWidget {
   });
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
     final expanded = expandedChild;
     final center = centerChild;
+    final isChatNG = ref.watch(isActiveProvider(LabsFeature.chatNG));
+    final roomRoute = isChatNG ? Routes.chatNGRoom.name : Routes.chatroom.name;
+    print('$isChatNG');
```

Comment:
a) print in production
b) this is what I wanted to avoid specifically... The widgets _should not_ have to figure out using different routes, but use the same route and then _the router_ decided were to go to ...